docs/index.asciidoc

```diff
@@ -47,10 +47,10 @@ For example, you can use the following log4j.properties configuration to write d
 
 Configuring log4j.properties in more detail is outside the scope of this migration guide.
 
-.Configuring filebeat
+===== Configuring filebeat
 
 Next,
-https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-installation.html[install
+https://www.elastic.co/guide/en/beats/filebeat/{branch}/filebeat-installation-configuration.html[install
```

Comment:
I've not seen `{branch}` used in plugin docs before - is this now recommended for links in plugin docs over `current`?

Reply:
When you see notations like `{logstash-ref}/plugins-inputs-http.html[input-http]`, the {branch} attribute is already embedded in the `{logstash-ref}` attribute.  

Using a variable gives us more flexibility because we can assign a value to variable like here: https://github.com/elastic/logstash-docs/pull/535.  Setting the `{branch}=current` is not perfect for the Versioned Plugin Reference, but it's a step above hardcoding it.